ocpp/routing.py

```diff
@@ -1,5 +1,6 @@
 import functools
 
+routables = []
```

Comment:
The rest of the code use the term 'route'. For example 'create_route_map' and `ChargePoint.route_map`. Can you use that term here as well instead of `routables`?

Reply:
my idea with 'routables' was that it was something that can be routed to, didn't want to use the term 'route' since it is only a text string. But if you want me to change to 'routes' I can to that

Reply:
I see your point. Can you leave it as is.